library/src/main/java/com/devbrackets/android/exomedia/core/video/scale/MatrixManager.java

```diff
@@ -97,7 +97,9 @@ protected void applyCenterCrop(@NonNull TextureView view, @NonNull Matrix transf
         float yScale = (float)view.getHeight() / intrinsicVideoSize.y;
 
         float scale = Math.max(xScale, yScale);
-        applyScale(view, transformMatrix, scale, scale);
+        xScale = scale / xScale;
```

Comment:
It should be a separate PR.  To me it still seems wrong, but it has been a while since I've done manual matrix multiplication so maybe I'm remembering something incorrectly.